### ISSUE TITLE
Tutorials

### ISSUE THREAD

**Issue by ScriptLineStudios** [pygame-community/pyga.me#11]
we can definitely work from here - https://www.pygame.org/wiki/tutorials although it would also be nice to write out own tutorials. Maybe we could write a mini tutorial series which takes you from start to finish building a simple game

**Comment by novialriptide** [comment]
I'm all for a written tutorial series. We might have them as Markdown files and compile them into HTML files later so anyone can contribute easily.

**Comment by itzpr3d4t0r** [comment]
Would be cool to have some way to run the example code in the web.

**Comment by novialriptide** [comment]
> Would be cool to have some way to run the example code in the web.

Yup! That's the plan for the far future. (Probably during the summer break when I don't have discrete math killing my GPA)

**Comment by simplyrohan** [comment]
For a while now, I've been thinking of a complete rewrite of the documentation, but with an approach similar to Flask. What I mean by this is that Flask teaches their documentation through the project `flaskr`. This is actually really helpful because it shows not only the technicalities but usage as well. By that point, tutorials could just smoothly walk right in. Still developing a game, tutorials could further push the "usage" docs by implementing higher-level concepts like physics. Another benefit of this approach is that instead of teaching concepts on their own, this also shows how to integrate features together to make a fully functional game. Finally, this type of teaching also could work excellently as a reference and not just a follow-along, baby-steps tutorial. My main point here is that documentation and tutorials could be integrated smoothly with a project to work on (I'm thinking of a platformer).
